### PR TITLE
Update google.js

### PR DESCRIPTION
**Pull request by AdhyanDhull**
Fixed linting errors

## What did you implement:

Closes #XXXXX

## Todos:

- [ ] Write tests
- [ ] Write documentation
- [ ] Fix linting errors
- [ ] Make sure code coverage hasn't dropped
- [ ] Provide verification config / commands / resources
- [ ] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** NO
***Is it a breaking change?:*** NO

### CODE REVIEW

File: test/storage/google.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 const nodeCloud = require('../../lib');
+const assert = chai.assert;
```
Comment (rajikaimal) [inline test/storage/google.js]: You should use single quotes. Please refer the style guide from Airbnb.

https://github.com/airbnb/javascript#modules

**Review by rajikaimal — CHANGES_REQUESTED** [review]

File: test/storage/google.js (hunk 1 above)
Comment (AdhyanDhull) [reply test/storage/google.js]: Kindly check now.

Comment (rajikaimal) [reply test/storage/google.js]: Looks good !

**Review by rajikaimal — APPROVED** [review]

**Comment by rajikaimal** [comment]
Thanks ! @AdhyanDhull

**Pull request merged**